lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/util/JsonUtils.java

```diff
@@ -166,6 +168,9 @@ public static JsonFactoryBuilder jsonFactoryBuilder()
                         .maxNestingDepth(Integer.MAX_VALUE)
                         .maxNumberLength(Integer.MAX_VALUE)
                         .build())
+                .enable(StreamReadFeature.USE_FAST_BIG_NUMBER_PARSER)
+                .enable(StreamReadFeature.USE_FAST_DOUBLE_PARSER)
+                .enable(StreamWriteFeature.USE_FAST_DOUBLE_WRITER)
```

Comment:
what is that and why is that not enabled by default; looks like something you would want to have so I guess there is some glitch.

Reply:
`This setting is disabled by default for backwards compatibility.`

Reply:
Is that behaviorally different? In what way it can break backward compatibility?

Reply:
No, it's enabled in different places already